src/redux/actions/authActions.js

```diff
@@ -1 +1,35 @@
-import * as types from '../types';
+import { LOGIN_SUCCESS, LOGIN_FAIL, LOGOUT, SET_LOADING } from '../types';
+import axios from 'axios';
+
+//Login User
+export const loginUser = accessToken => async dispatch => {
+  try {
+    setLoading();
+    const res = await axios.post('http://127.0.0.1:8000/google/auth/token/', {
```

Comment:
try setting up the `proxy` settings for the base url so that we don't have to type down the whole url every time.

Reply:
Okay.